beacon-chain/rpc/service.go

```diff
@@ -182,15 +173,16 @@ func (s *Service) Start() {
 		beaconDB:         s.beaconDB,
 		operationService: s.operationService,
 		p2p:              s.p2p,
-		chainService:     s.chainService,
+		attReceiver:      s.chainService,
+		headRetriever:    s.chainService,
 		cache:            cache.NewAttestationCache(),
 	}
 	validatorServer := &ValidatorServer{
 		ctx:                s.ctx,
 		beaconDB:           s.beaconDB,
```

Comment:
you need to also intialize headFetcher here , maybe change it to headRetriever to make it consistent with the other services